RecoTracker/TkSeedingLayers/src/SeedingLayerSetsBuilder.cc

```diff
@@ -185,7 +186,7 @@ SeedingLayerSetsBuilder::SeedingLayerSetsBuilder(const edm::ParameterSet & cfg,
 {
   std::vector<std::string> namesPset = cfg.getParameter<std::vector<std::string> >("layerList");
   std::vector<std::vector<std::string> > layerNamesInSets = this->layerNamesInSets(namesPset);
-
+  fastSimrecHitsToken_ = iC.consumes<FastTrackerRecHitCollection>(edm::InputTag("fastTrackerRecHits")); //fix: to pass as argument to constructor
```

Comment:
(This is probably on your list already, but to not forget) please move this to a FastSim-specific constructor which takes the `edm::InputTag` as an argument as discussed in https://github.com/cms-sw/cmssw/pull/23363#discussion_r192752766